tests/modules/picard/crosscheckfingerprints/main.nf

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PICARD_CROSSCHECKFINGERPRINTS } from '../../../../modules/picard/crosscheckfingerprints/main.nf'
+
+process STUB_HAPLOTYPE_MAP {
+
+    output:
+    path "haplotype_map.txt" ,emit:haplotype_map
+
+    stub:
+    """
+    echo haplotype_map > haplotype_map.txt
+    """
+}
```

Comment:
For the stub run you can just declare a channel empty for the `haplotype_map` input I think, as shown in the next  suggestion
```suggestion
```

Reply:
Can't do this, the haplotype map is a mandatory argument for this tool